languages/csharp/exercises/concept/README.md

```diff
@@ -4,29 +4,9 @@ The concept exercises are based on this [list of concepts][reference-shared].
-| [`enums`][concept-exercise-enums]                                   | `enums`<br/>`pattern-matching-constants`                             | `conditionals`<br/>`strings`                                                      |
-| [`flag-enums`][concept-exercise-flag-enums]                         | `bit-manipulation`<br/>`flag-enums`                                  | `attributes`</br>`enums`<br/>`integers`                                           |
-| [`floating-point-numbers`][concept-exercise-floating-point-numbers] | `floating-point-numbers`<br/>`while-loops`                           | `conditionals`<br/>`numbers`                                                      |
-| [`inheritance`][concept-exercise-inheritance]                       | `inheritance`                                                        | `booleans`<br/>`classes`<br/>`conditionals`<br/>`constructors`<br/>`strings`      |
-| [`method-overloading`][concept-exercise-method-overloading]         | `method-overloading`<br/>`named-arguments`<br/>`optional-parameters` | `classes`<br/>`constructors`<br/>`enums`<br/>`properties`<br/>`strings`           |
-| [`nullability`][concept-exercise-nullability]                       | `nullability`                                                        | `conditionals`<br/>`memory-allocation`<br/>`strings`                              |
-| [`numbers`][concept-exercise-numbers]                               | `conditionals`<br/>`numbers`                                         | `basics`                                                                          |
-| [`properties`][concept-exercise-properties]                         | `properties`                                                         | `classes`<br/>`enums`<br/>`exceptions`<br/>`floating-point-numbers`<br/>`numbers` |
-| [`strings`][concept-exercise-strings]                               | `strings`                                                            | `basics`                                                                          |
-
-It's only important that it's reasonably easy to _find_ the exercise. It's okay if the name isn't perfect. We **will** iterate on this.
+If you need to see a list of the completed exercises the follow this [link][config-json] to the C# track's _config.json_ file.  In addtion, in the file, you can search for _concepts_ to ascertain which exercises they are featured in and the _prerequisites_  sections will show the relationships between the exercises.
+
+[config.json][config-json] should be updated, as part of the PR, as the last stage in implementing an exercise.  Note that you will need to generate and include an UUID to identify the exercise.  This also applies when improving an exercise if you modify the set of concepts or prerequisites.
```

Comment:
Maybe instead of listing the UUID requirement, we could link to the https://v3.exercism.io/csharp/contributing link to get people on the right track to get started? They don't have to worry about the UUID until the end of the PR process.

```suggestion
The [`config.json` file][config-json] should be updated, as part of the PR, as the last stage in implementing an exercise. This also applies when improving an exercise if you modify the set of concepts or prerequisites.
```